### DISCUSSION TITLE
修改评论区默认文字无效

### DISCUSSION THREAD

**Discussion by FHR2048** [walinejs/waline#1922] [Q&A]
hexo-theme-next
配置项:
......
waline:
  placeholder: xxxxx #评论框的默认文字
......

但始终显示同一文字:
![如图](https://cdn.staticaly.com/gh/FHR2048/TuChuang/main/1.jpeg)

**Comment by FHR2048** [comment]
`<script class="next-config" data-name="waline" type="application/json">{"lang":"zh-cn","enable":true,"serverURL":"fhr-comments.zeabur.app","cssUrl":"https://unpkg.com/@waline/client@v2/dist/waline.css","commentCount":true,"pageview":false,"dark":"body.darkmode--activated","reaction":false,"placeholder":"xxxxxxxx","avatar":"mm","meta":     ["nick","mail","link"],"pageSize":10,"visitor":false,"comment_count":true,"requiredFields":[],"libUrl":"https://unpkg.com/@waline/client@2.14.9/dist/waline.js","el":"#waline","comment":true,"path":"/"}</script>
`
并没有关于欢迎评论的内容,所以它到底是哪来的？？？

**Comment by lizheming** [comment]
https://waline.js.org/cookbook/customize/locale.html

**Reply by FHR2048** [reply]
对于hexo-theme-next来说,修改这个默认文字应该就是：
placeholder: xxxxxxxx
但更改就是无效的

[https://waline.js.org/cookbook/customize/locale.html#%E4%BE%8B%E5%AD%90](https://waline.js.org/cookbook/customize/locale.html#%E4%BE%8B%E5%AD%90)

**Comment by FHR2048** [comment]
emm,这个貌似是中文默认显示文字,那我该怎么更改它？
